index.js

```diff
@@ -1,9 +1,302 @@
 // Toogle Mobile Menu
-const menu = document.querySelector(".mobileMenu");
-const mobileMenu = document.querySelector(".menu-mobile");
+const menu = document.querySelector('.mobileMenu');
+const mobileMenu = document.querySelector('.menu-mobile');
 const toggleMenu = () => {
-  mobileMenu.classList.toggle("showMenu");
+  mobileMenu.classList.toggle('showMenu');
 };
 
-menu.addEventListener("click", () => toggleMenu());
-mobileMenu.addEventListener("click", () => toggleMenu());
\ No newline at end of file
+menu.addEventListener('click', () => toggleMenu());
+mobileMenu.addEventListener('click', () => toggleMenu());
+
+// Toogle pop Up Menu
+
+const projects = [
+  {
+    id: 1,
```

Comment:
- [ ] nice job. Please make your background a little bit darker/blurer when the desktop popup comes up(especially on the white sections) so the start and end of the popup can be seen. 
 - [ ] please reduce the size of the tech icons on the popup so the match the figma design
yours
![Screenshot (870)_LI](https://user-images.githubusercontent.com/65298048/223467728-fe601aa5-adf8-4d38-82ab-de00613eec54.jpg)
figma
![Screenshot (871)](https://user-images.githubusercontent.com/65298048/223467783-b2736481-c311-47f3-bfe8-87cc1a8ffdf6.png)